src/mender/client/inventory.py

```diff
@@ -51,7 +51,6 @@ def request(
         log.error(f"Failed to upload the inventory: {e}")
         return None
     log.debug(f"inventory response: {r}")
-    log.error(f"Error {r.reason}. code: {r.status_code}")
     if r.status_code != 200:
         log.error(f"{r.json()}")
```

Comment:
Once you removed the above one, maybe add here some prefix for context.
```suggestion
        log.error(f"Got response {r.json()}")
```

Reply:
Could I just have it in a separate commit? :pray:

Reply:
sure! and sure! fixed to `        log.error(f"Got inventory response: {r.json()}")` and [here is the commit](https://github.com/mendersoftware/mender-python-client/pull/8/commits/e1ab143bdf9aeeea88da1fea34a904391d292847)